### PR TITLE
Release 2018.9

### PR DESCRIPTION
**Pull request by uajain**

### CODE REVIEW

**Comment by jlebon** [comment]
Yeah, I suppose it's about that time! There's #1760 that seems low risk, and I'd like to get #1740 into this release as well if possible.

**Comment by mwleeds** [comment]
This will have to be redone if https://github.com/ostreedev/ostree/pull/1763 is merged. Sorry about that.

**Comment by uajain** [comment]
> This will have to be redone if #1763 is merged. Sorry about that.

Looks like it's approved but not merged due to some internal error. I'll check again after some time.
@jlebon @cgwalters Do you have anything else(symbol-wise) before I update this release PR?

**Comment by cgwalters** [comment]
> @jlebon @cgwalters Do you have anything else(symbol-wise) before I update this release PR?

Nothing from me.

**Comment by uajain** [comment]
Updated the PR.

**Comment by cgwalters** [comment]
Thanks.  @uajain what time zone are you in?  Often jlebon and I set up an etherpad where we write release notes together.  Both of us are in EST.  Do you want to participate in that too?  If so, would say 2-3pm EST tomorrow work?

**Comment by uajain** [comment]
Hey, @cgwalters I am in India (UTC +5:30) timezone, so I don't think that time suits me. However, I am happy to get involved asynchronously for any matter.

**Comment by cgwalters** [comment]
Hm, some test flakes there.  But let's do this:

@rh-atomic-bot r+ b0eedc9

We can get the release commit merged, and I'll post a link to an etherpad here for us to collaborate on release notes and then take care of pushing the tag.

**Comment by jlebon** [comment]
Fix in #1765.
Actually, maybe I should just inline it here. Will do that.

File: src/ostree/ot-main.c [hunk 1]
```diff
@@ -354,7 +354,7 @@ ostree_option_context_parse (GOptionContext *context,
        * possible to parse */
       g_auto(GStrv) features = g_strsplit (OSTREE_FEATURES, " ", -1);
       g_print ("%s:\n", PACKAGE_NAME);
-      g_print (" Version: %s\n", PACKAGE_VERSION);
+      g_print (" Version: v%s\n", PACKAGE_VERSION);
```
Comment (cgwalters) [inline src/ostree/ot-main.c]: How about making this `g_print (" Version: '%s'\n", PACKAGE_VERSION);` instead?

Comment (jlebon) [reply src/ostree/ot-main.c]: Done! :arrow_down:

**Comment by cgwalters** [comment]
@rh-atomic-bot r+

**Comment by cgwalters** [comment]
Hmm, although the commit is after the release commit right?  Probably need to rebase it to before?

**Comment by jlebon** [comment]
We could do that. Though note whatever out there was relying on this before should just keep working in this release because `2018.9` still parses fine. So it doesn't *have* to be in this release.

**Comment by cgwalters** [comment]
I didn't start on the rest of release yet (relnotes/tag etc)...will look at this tomorrow.

**Comment by cgwalters** [comment]
Set up an etherpad for this release: https://public.etherpad-mozilla.org/p/ostree-2018.9

**Pull request closed without merging**